create_jenkins_job.py

```diff
@@ -149,9 +148,7 @@ def create_job(os_name, job_name, template_file, additional_dict):
         create_job(os_name, 'ci_packaging_' + os_name, 'packaging_job.xml.em', {
             'cmake_build_type': 'RelWithDebInfo',
             'test_bridge_default': 'true',
-            'use_fastrtps_default': 'true',
-            'use_opensplice_default': 'true',
-            'use_connext_default': 'false' if os_name is 'linux-aarch64' else 'true',
+            'ignore_rmw_default': {'rmw_connext_cpp', 'rmw_connext_dynamic_cpp'} if os_name is 'linux-aarch64' else set(),
```

Comment:
the empty set here and elsewhere is going to cause connext_dynamic to be enabled on our jobs

Reply:
@dirk-thomas (following on from your point in https://github.com/ros2/ci/pull/212#issuecomment-407955702 that the builds will pass because of an AMENT_IGNORE): the checkbox will still be selected on all of those jobs, so the default for the checkbox should be set to false to not be misleading (or the description of the checkbox  and the build summaries should be updated to mention that)

Reply:
I don't think these two parts should be coupled. The decision to place the marker file in the source repo is one thing. Updating the logic in this repo to "duplicate" that is not necessary.

The inverse is already the case if we drop another marker file in any of the RMW repos that won't affect this repo.

Reply:
the difference is that we would not put a marker in an rmw implementation that we want enabled by default. we want connext dynamic disabled by default on our CI (whether there's an AMENT_IGNORE marker there for users building from source or not), and this should be indicated in the defaults for the jobs.

Reply:
I don't appreciate that changes that are requested become the responsibility of the reviewer to implement. If you can get consensus from others that having the checkbox enabled by default for an unsupported implementation is reasonable, then I'd be happy to reconsider my position, but otherwise, as I see it, the code can't be approved in its current state

Reply:
Since this came up in the waffle meeting as an "RFC"...

I agree with @dhood, it's confusing to have "connext dynamic not ignored" in CI but it still not build because of the file in the source that disables it. I understand why both exist, but until we move to reenable connext dynamic I'd say it is less confusing to have it disable by default in CI too.

The other option would be to comment out or just remove the check box for connext dynamic until we reenable it, because if I understand correctly, if we default to off it will still not work if the user manually flips the switch to "not ignored", which is pretty misleading too. If the state of the check mark doesn't matter, then we should probably just remove it for now.

> I don't appreciate that changes that are requested become the responsibility of the reviewer to implement.

I tend to agree, depending on the scale of the change requested. From my perspective, this one is small but there is a disagreement about whether or not to do it, and in that case asking the reviewer to do because you don't agree it should be done (not because it's technically difficult) is not the right response (imo). But we seem to disagree about that implicitly quite often.